server/routes/search.js

```diff
@@ -0,0 +1,36 @@
+require("dotenv").config();
+
+const express = require("express");
+const { Op } = require("sequelize");
+const router = express.Router();
+const db = require("../models");
+const authenticateToken = require("./helper/authenticateToken");
+router.use(express.json());
+
+router.get("/search", authenticateToken, (req, res) => {
+  let serachTerms = req.query;
```

Comment:
Looks good! except theres a typo here -- serach ... it still works bc the typo is repeated on the next couple lines but if you've got time  can you fix that?

Reply:
fixed!